cmd/collector/app/handler_test.go

```diff
@@ -48,9 +49,17 @@ func (p *possiblyErroringJaegerBatchesHandler) SubmitBatches(ctx tchanThrift.Con
 	return nil, p.err
 }
 
+type possiblyErroringZipkinBatchesHandler struct {
```

Comment:
I was bing consistent with `possiblyErroringJaegerBatchesHandler` on link 44. I can update this to `mockZipkinHandler` if you prefer.